http/swagger.yml

```diff
@@ -8006,6 +8007,9 @@ components:
           type: string
         shadeBelow:
           type: boolean
+        position:
+          type: string
+          enum: [overlaid, stacked]
```

Comment:
i know that that this pattern has already been established, but out of curiosity, is it possible to create a `LinePositioning` type in one place and use that pre-defined type elsewhere? That way, if we need to change the positioning enum by adding another type of positioning, we only need to make a change in one place instead of tracking down all the possible places we have this enum defined. not sure if that's possible though

Reply:
Golang has enums but I don't think this is possible inside a swagger.yml file just yet. There is a proposal for this. See https://github.com/go-swagger/go-swagger/pull/744